pkgs/development/python-modules/autograd-gamma/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildPythonPackage, fetchFromGitHub, autograd, pytest, scipy }:
+
+buildPythonPackage rec {
+  pname = "autograd-gamma";
+  version = "0.4.1";
```

Comment:
```suggestion
  version = "0.4.1";
  disabled = isPy27;
```
some of the syntax isn't compatible with python2.
```
warning: ignoring the user-specified setting 'sandbox', because it is a restricted setting and you are not a trusted user
builder for '/nix/store/zw4pdpd4jgc4xynjk42w5p2i2p9ijscp-python2.7-autograd-gamma-0.4.1.drv' failed with exit code 2; last 10 log lines:
  /nix/store/amv9br0s7xxyr1qipqjqbss5jpdcvha7-python2.7-pytest-4.6.8/lib/python2.7/site-packages/_pytest/assertion/rewrite.py:304: in load_module
      exec(co, mod.__dict__)
  tests/test_autograd_gamma.py:8: in <module>
      from autograd_gamma import gammainc, gammaincc, betainc
  E     File "/nix/store/xgg1rh2dr8h78fx554fa50b8k1bn9dbx-python2.7-autograd-gamma-0.4.1/lib/python2.7/site-packages/autograd_gamma/__init__.py", line 47
  E       new_f = lambda x, *args: f(*args[:argnum], x, *args[argnum:])
  E                                                     ^
  E   SyntaxError: invalid syntax
  !!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!
  =========================== 1 error in 0.83 seconds ============================
cannot build derivation '/nix/store/97plly7pbhl6hvrbb8a7j7f0r6vsinwd-env.drv': 1 dependencies couldn't be built
```